src/sgame/sg_utils.cpp

```diff
@@ -162,7 +162,7 @@ int G_ShaderIndex( const char *name )
 
 	if ( !i )
 	{
-		Log::Warn( "Missing shader: %s", name );
```

Comment:
"Missing shader nullptr" is not very actionable for the user...

Reply:
yes 😅️

Well, this is mostly to not crash the game while leaving an useful message for the developper, I doubt an user-provided asset is expected to produce a nullptr by itself, a code has to do it.

Reply:
Well, this PR fixes a regression in current master that can crash the game, I'll merge this as is. I don't think we need something “better”, because a more literary wording would just be less meaningful, and I see no reason to complicate more the code by adding more tests and alternate error messages. If this messages prints there is a nullptr, that's a message for the developer, and I want user provides logs that are explicit to developers.